all.json

```diff
@@ -97,6 +97,7 @@
     "claimpolkadot.org",
     "coinbase-ether.com",
     "coinrau.com",
+    "coinoneswap.finance",
```

Comment:
This is one of those I cannot evaluate - it is not part of the Polkadot ecosystem, so it refers to items foreign (to the ecosystem) wallet connections and tokens.